src/pages/Sphere/hooks/useStakingPool.ts

```diff
@@ -0,0 +1,18 @@
+import { useQuery } from '@tanstack/react-query';
+import { useCyberClient } from 'src/contexts/queryCyberClient';
+
+function useStakingPool() {
```

Comment:
I think better place such hooks to `features/{staking}`, to match SDK structure. and it may be reused later